L/LibOSXUnwind/build_tarballs.jl

```diff
@@ -1,11 +1,11 @@
 using BinaryBuilder
 
 name = "LibOSXUnwind"
-version = v"0.0.6"
+version = v"0.0.7"
 
 # Collection of sources required to build libosxunwind
 sources = [
-    ArchiveSource("https://github.com/JuliaLang/libosxunwind/archive/v$(version).tar.gz",
+    ArchiveSource("https://github.com/JuliaLang/libosxunwind/archive/v0.0.6.tar.gz",
```

Comment:
This is not so nice; alternatives include: add a 0.0.7 tag to that repository; or switch to `GitSource` with a fixed sha1 (but is that better?)

Reply:
This is fine; it's how a lot of our stuff is going to be built in the future.  The JLL version will be more independent from the upstream source version. :(

Reply:
Indeed, good point.